### ISSUE TITLE
The create new action interface is a bit small

### ISSUE THREAD

**Issue by LuoiZQ** [rednblackgames/HyperLap2D#83]
The create new action interface is a bit small, and worse, the view cannot be moved. The same problem exists in the main interface.

**Comment by fgnm** [comment]
Right, I've in mind to rework from scratch the whole node graph editor and extend it for other things too.. Just hope to find the time to make all those things soon :)
